RELEASES.md

```diff
@@ -1,3 +1,71 @@
+Version 1.22.0 (2017-11-23)
+==========================
+
+Language
+--------
+- [`non_snake_case` lint now allows extern no-mangle functions][44966]
+- [Now accepts underscores in unicode escapes][43716]
+- [`#![feature(const_fn)]` is now no longer required for
+  calling const functions.][43017] It's Still required for creating
+  constant functions.
+
+Compiler
+--------
+- [rustc now defaults to having 16 codegen units at debug on supported platforms.][44853]
+- [rustc will no longer inline in codegen units when compiling for debug][45075]
+  This should decrease compile times for debug builds.
+- [strict memory alignment now enabled on ARMv6][45094]
+- [Remove support for the PNaCl target `le32-unknown-nacl`][45041]
+
+Libraries
+---------
+- [Allow atomic operations up to 32 bits
+  on `armv5te_unknown_linux_gnueabi`][44978]
```

Comment:
GitHub isn't rendering this link -- perhaps it can't be multiline?

Reply:
I can't replicate this. It's a working link for me. Are you getting still getting this?

Reply:
Looks fine now. 🤷‍♂️